locals.tf

```diff
@@ -4,6 +4,14 @@ locals {
     (contains(["rapid", "stable", "node-image"], var.automatic_upgrade_channel) && var.kubernetes_version == null && var.default_node_pool.orchestrator_version == null)
   )
   dns_prefix = coalesce(var.dns_prefix, random_string.dns_prefix.result)
+  kube_admin_enabled = (
+    !var.local_account_disabled
```

Comment:
According to the [variable's declaration] (https://github.com/Azure/terraform-azurerm-avm-res-containerservice-managedcluster/blob/e1ecf513b76a70ccc45e98d489176f3df6476f57/variables.tf#L448-L452), `var.local_account_disabled` could be `null`, a `var.local_account_disabled == false` here?

Reply:
Could you clarify this comment? It's a bool - either true or false - should never be null